src/lib/types/array.js

```diff
@@ -63,13 +63,13 @@ function arrayType(value, path, resolve, traverseCallback) {
 
   if (optionAPI('minItems')) {
     // fix boundaries
-    minItems = !maxItems
+    minItems = maxItems === undefined
```

Comment:
I think could test like `typeof maxItems  ===  'undefined'` instead

Reply:
Done in 65c9366